src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -124,7 +130,9 @@ public void draw(final Rectangle bounds, final GameData data, final Graphics2D g
           break;
       }
     } else{
-      drawUnit(graphics, img.get(), placementPoint, bounds);
+      if(img.isPresent()) {
+        drawUnit(graphics, img.get(), placementPoint, bounds);
+      }
```

Comment:
This fixes a bad assumption that the image is always there. To allow the game to keep running, we effectively ignore and do not draw the image. The logging above will have taken place, so there is no logging needed